### ISSUE TITLE
Improve labels of pop-up window

### ISSUE THREAD

**Issue by AndreasSahlberg** [kartoza/least-cost-electrification-somalia#36]
From the WB team: "For the pop-up window, clarify whether those numbers including those already electrified in the previous year, or only new connections." 

@AndreasSahlberg to provide clarification. 
@meomancer and @meomancer, do you know where to update the labels?

**Comment by meomancer** [comment]
Could you provide the example of the data that needs to be show on popup?
The popup is hardcoded, so need to implement it

**Comment by AndreasSahlberg** [comment]
I think it should show the something like following:
Details
TOTAL POPULATION 2030 (Column "Pop2030")
30000
TOTAL HOUSEHOLDS 2030 (Column "Households2030 - new column to be added to the result files)
5000
NEW PEOPLE CONNECTED 2020-2025 (Column "NewConnections2025") 
10000
NEW PEOPLE CONNECTED 2025-2030 (Column "NewConnections2030")
20000
INVESTMENT REQUIRED 2020-2025 (Column "InvestmentCost2025")
$ 4 072 241
INVESTMENT REQUIRED 2025-2030 (Column "InvestmentCost2030")
$ 9 083 241
ADDED CAPACITY 2020-2025 (Column "NewCapacity2025")
4 MW
ADDED CAPACITY 2025-2030 (Column "NewCapacity2030")
8 MW

Or if that does not display nicely on the platform, it should show for the time-step:

TOTAL POPULATION
30000
TOTAL HOUSEHOLDS
5000
NEW PEOPLE CONNECTED IN TIMESTEP
10000
INVESTMENT REQUIRED IN TIMESTEP
$ 4 072 241
ADDED CAPACITY IN TIMESTEP
4 MW

![image](https://user-images.githubusercontent.com/35484968/107412287-ccd31880-6b0f-11eb-90d3-2bd1d03bb62f.png)

**Comment by meomancer** [comment]
hi @AndreasSahlberg 
The PR:
https://github.com/kartoza/global-electrification-platform-explorer/pull/9

Households is still empty

![Selection_070](https://user-images.githubusercontent.com/4530905/107482844-d06bab80-6bb2-11eb-862f-0a373e7a7252.png)
![Selection_071](https://user-images.githubusercontent.com/4530905/107482852-d2356f00-6bb2-11eb-930b-2cf81f2e6daa.png)

@AndreasSahlberg 
For additional field that needs to show on popup, for example in here is **NewConnections**, we need to put it as filters. Because the backend of gep just saves data into database for the columns that are in filters 
Except [Pop, PopConnected, ElecCode, FinalElecCode, InvestmentCost, NewCapacity, ElecStatusIn]

So basically if we want to show the new column into popup that is not in the above list, we need to add new filter that using the column.
And also for now, it is just working on timestep column.

Do you have any feedback?

**Comment by AndreasSahlberg** [comment]
@meomancer please add "TOTAL BUILDINGS" with the 'Buildings2025' and 'Buildings2030' columns available in the new results (**updated_results.zip**) available here instead of TOTAL HOUSEHOLDS:
https://drive.google.com/drive/folders/15quQoRmSAf46Duo_DUEevRO6KXffChW3

Thanks!

**Comment by AndreasSahlberg** [comment]
Looks good!
